downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,6 +60,20 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
+- Introduced the `Extract to transform function` code action to extract a value expression into a transform function.
```

Comment:
Do we generally use the term "transform function"?

Reply:
These function templates are used for the data mapper, and the code action is employed to enhance the DX in this context. While there is no coined term for such functions, we decided to use the transform function considering its use case. https://github.com/ballerina-platform/ballerina-lang/pull/42194

Reply:
Initially, we chose this name to uniquely identify these function templates, distinguishing that this code action is not applicable for other scenarios. However, after some consideration, I am also fine with using `expression-bodied function` or `arrow function`. Since the LS restricts the code action only to the given context, we may not need to be definitive with the code action name.

Looping in @KavinduZoysa 

If we were to change the release note here, we would need to send a PR to `ballerina-lang` with the respective name change.

Reply:
Let's keep as is for now since we may need a discussion with a wider group before changing this. If we eventually change it, we can mention it the relevant release note and say the name of the CA changed.